### PR TITLE
DM-23193: outputs more data for ATDome

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: sal_interfaces/ATDome/ATDome_Events.xml [hunk 1]
```diff
@@ -153,6 +153,55 @@
...
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+    </SALEvent>
+    <SALEvent>
+        <Subsystem>ATDome</Subsystem>
+        <Version>3.8.2</Version>
+        <EFDB_Topic>ATDome_logevent_lastAzimuthGoTo</EFDB_Topic>
+        <Alias>lastAzimuthGoTo</Alias>
+        <Description>Probably the last commanded azimuth position. The value reported in full status as `Last Azimuth GoTo`.</Description>
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply sal_interfaces/ATDome/ATDome_Events.xml]: I confirmed that this is the last commanded position.

File: sal_interfaces/ATDome/ATDome_Events.xml [hunk 2]
```diff
@@ -245,6 +301,34 @@
...
+        <item>
+            <EFDB_Name>azimuthMoveTimeout</EFDB_Name>
+            <Description>Timeout for azimuth moves.</Description>
+            <IDL_Type>float</IDL_Type>
+            <Units>second</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>doorMoveTimeout</EFDB_Name>
+            <Description>Timeout for opening or closing (what? either shutter door?).</Description>
```
Comment (patrickingraham) [inline sal_interfaces/ATDome/ATDome_Events.xml]: This is how long it waits to fully close the shutters before timing out.

File: sal_interfaces/ATDome/ATDome_Events.xml [hunk 3]
```diff
@@ -231,6 +280,13 @@
             <Units>second</Units>
             <Count>1</Count>
         </item>
+        <item>
+            <EFDB_Name>dropoutTimer</EFDB_Name>
+            <Description>What is this? How does it differ from doorMoveTimeout?</Description>
```
Comment (patrickingraham) [inline sal_interfaces/ATDome/ATDome_Events.xml]: I must be blind but I don't know where you're seeing this....

Comment (r-owen) [reply sal_interfaces/ATDome/ATDome_Events.xml]: Here is sample full status that Tiago gave me:
```
MAIN AJAR 000
DROP AJAR 000
[OFF] 00
POSN 22.90
RR 000
Emergency Stop Active: 0
Top Comm Link OK: 0
Home Azimuth:  0.00
High Speed (degrees): 5.00
Coast (degrees): 0.50
Tolerance (degrees): 1.00
Encoder Counts per 360: 4018143232
Encoder Counts: 81630126151
Last Azimuth GoTo: 20.00
Azimuth Move Timeout (secs): 120
Rain-Snow enabled: 0
Cloud Sensor enabled: 1
Watchdog Reset Time: 600
Dropout Timer: 5
Reverse Delay: 5
Main Door Encoder Closed: 0
Main Door Encoder Opened: 0
Dropout Encoder Closed: 0
Dropout Encoder Opened: 0
Door Move Timeout (secs): 10
```
Notice that there are two door timeouts: Door Move Timeout and Dropout Timer. The manual, of course, gives no hint about what they mean.

See also the EUI. There is a Dropout Timer near the bottom of the TopConf section, in addition to a Door Move Timeout near the top.

File: sal_interfaces/ATDome/ATDome_Events.xml (hunk 2 above)
Comment (r-owen) [reply sal_interfaces/ATDome/ATDome_Events.xml]: Only close, not open?

File: sal_interfaces/ATDome/ATDome_Events.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (r-owen) [reply sal_interfaces/ATDome/ATDome_Events.xml]: Thank you. I will simplify the text. I'm still wondering what it is set to when ATDome has not been homed, or when it is being homed, but I'm not going to fret about it.

**Pull request merged**